server/src/handlers/auth_handler.rs

```diff
@@ -215,6 +215,29 @@ pub async fn create_account(
                 })?
         }
     };
+    let org_plan_sub = get_organization_by_id_query(org.id, pool.clone())
+        .await
+        .map_err(|_| {
+            ServiceError::InternalServerError("Could not find organization for user".to_string())
+        })?;
+    let user_org_count_pool = pool.clone();
+    let user_org_count = web::block(move || get_user_org_count(user_id, user_org_count_pool))
+        .await
+        .map_err(|_| {
+            ServiceError::InternalServerError("Blocking error getting org user count".to_string())
+        })?
+        .map_err(|err| ServiceError::InternalServerError(err.message.to_string()))?;
+    if user_org_count
```

Comment:
Should be user_org_count + 1 bc we  have to check if an additional account would cause plan overflow